### ISSUE TITLE
Doesn't update $DARKMODE when appearance is changed with Shortcuts

### ISSUE THREAD

**Issue by gregorias** [bouk/dark-mode-notify#10]
### TL;DR

If I change appearance using a shortcut defined in [Shortcuts](https://support.apple.com/guide/shortcuts-mac/intro-to-shortcuts-apdf22b0444c/mac), the DARKMODE variable doesn't update.

To reproduce, set the shortcut like so:

<img width="930" alt="image" src="https://github.com/bouk/dark-mode-notify/assets/722385/9d2a0666-4fe0-41e9-a389-8eefe6e771c2">

Try running the app and displaying the $DARKMODE variable.

### Expected behavior

`$DARKMODE` variable is updated accordingly.

### Observed behavior

`$DARKMODE` variable is retains its previous value and ignores system changes.

### Terraform Configuration

```hcl
I don't know how to do this.
```

### Terraform Version

```sh
I don't know how to do this.
```

**Comment by huyz** [comment]
Works for me if I use AppleScript. So this could be a workaround for you if you have your shortcut trigger an AppleScript:
```applescript
tell application "System Events"
    tell appearance preferences 
        set dark mode to not dark mode 
        if dark mode then
            set mode to "dark"
        else
            set mode to "light"
        end if
    end tell
end tell

```
